en/operator-precedence.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+language: 'en'
+version: '0.12'
+---
+# Operator Precedence
+**Operator precedence** determines how operators are parsed concerning each
+other. Operators with higher precedence become the operands of operators with
+lower precedence. For example, in the expression `1 + 5 * 3`, the answer is 16
+and not 18 because the multiplication (`*`) operator has a higher precedence
+than the addition (`+`) operator. Parentheses may be used to force precedence,
+if necessary. For instance: `(1 + 5) * 3` evaluates to 18.
```

Comment:
**Operator precedence** determines how operators are parsed. Operators with higher precedence become the operands of operators with lower precedence. For example, in the expression `1 + 5 * 3`, the answer is 16 and not 18 because the multiplication (`*`) operator has a higher precedence than the addition (`+`) operator. Parentheses may be used to force precedence, if necessary. For instance: `(1 + 5) * 3` evaluates to 18.